events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -95,43 +111,222 @@ <h3>Questions</h3>
                 }
             }
             function onScriptEventReceived(event) {
-                if (typeof event === "string") {
-                    if (event.indexOf("question") !== -1) {
+                if (event === "newGame") {
+                    databaseButton.disabled = true;
+                    typeButton.disabled = true;
+                    diffButton.disabled = true;
+                    catButton.disabled = true;
+                    beginButton.disabled = false;
+                    endButton.disabled = true;
+                    newQButton.disabled = true;
+                    showQButton.disabled = true;
+                    answerButton.disabled = true;
```

Comment:
using this many lines of code to disable something feels very fragile to me. it seems to me like each of these events correspond to application states in a state machine. if you modeled your application that way, you could cut down on the number of lines of code in here pretty drastically. it's probably not worth fixing at this point - please keep that in mind for the future.